app/lib/actions/katello/repository/remove_content.rb

```diff
@@ -33,25 +35,33 @@ def plan(repository, content_units, options = {})
             plan_action(pulp_action, :pulp_id => repository.pulp_id,
                                      :clauses => {:association => {'unit_id' => {'$in' => uuids}}
             })
-
-            plan_self(:repository_id => repository.id, :user_id => ::User.current.id,
-                      sync_capsule: sync_capsule)
+            plan_self
             plan_action(CapsuleGenerateAndSync, repository) if sync_capsule
           end
         end
 
+        def create_sub_plans
+          trigger(Actions::Katello::Repository::MetadataGenerate,
+                  ::Katello::Repository.find(input[:repository_id]))
+        end
+
+        def trigger(*args)
```

Comment:
Is it being called anywhere without args?  Why does it need to be overridden, as it seems like its the same basic method as defined in dynflow (aside from the `if args.any?` check)